lib/page/country.rb

```diff
@@ -0,0 +1,9 @@
+module Page
+  class Country
+    attr_reader :country, :title
+    def initialize(country, all_countries)
+      @country = all_countries.find { |c| c[:url] == country }
+      @title = "EveryPolitician: #{@country[:name]}" if @country
```

Comment:
`initialize` is doing a bit too much work here. I suspect I led you into this bad behaviour, though, with how `Page::Download` works, so I've cleaned that one up in https://github.com/everypolitician/viewer-sinatra/pull/14625